server/start/tryCatchForIndexPermissionError.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Wazuh app - React HOCs to manage the message when elastic show a Response error
+ * Copyright (C) 2015-2021 Wazuh, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * Find more information about this on the LICENSE file.
+ */
+import { log } from '../lib/logger';
+
+export const tryCatchForIndexPermissionError = (wazuhIndex: string, logLocation?: string, context?) => (functionToTryCatch) => async () => {
+    console.log("HOAL QUE PASA NENE", logLocation);
+    try {
+        await functionToTryCatch();
+    }
+    catch (error) {
+        switch(error.message){
+            case 'security_exception':
+              error.message = (((((error.meta || error.message).body || error.message).error || error.message).root_cause[0] || error.message).reason || error.message);
+              break;
+            case 'Response Error':
```

Comment:
I would pass this string 'Response Error' to constants.

Reply:
I have changed it, thank you!